core/modules/path/path.admin.inc

```diff
@@ -643,9 +643,15 @@ function path_bulk_update_form($form, &$form_state) {
       // vocabularies.
       if (isset($path_info['pattern items'])) {
         foreach ($path_info['pattern items'] as $item => $item_label) {
+          $human_name = $item;
+          if ($type == 'taxonomy_term') {
+            //For vocabularies, use their human name in the checkbox label instead of their 
+            //machine name.
+            $human_name = $path_info[$item . '-human'];
+          }
```

Comment:
There should be a space between each leading `//` and the beginning of the respective comment line.

Other than that, looks good 👍 ...if I was to suggest anything, perhaps this should be an `if`/`else`? ...or something like this?:

```
// For vocabularies, use their human name in the checkbox label instead of their 
// machine name.
$human_name = $type == 'taxonomy_term' ? $path_info[$item . '-human'] : $item;
```

Reply:
Thanks @klonos! I made the changes you suggested.